executor/disabled_optimize_list.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"context"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/expression"
+	plannercore "github.com/pingcap/tidb/planner/core"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/sqlexec"
+)
+
+// ReloadDisabledOptimizeListExec indicates ReloadDisabledOptimizeList executor.
+type ReloadDisabledOptimizeListExec struct {
+	baseExecutor
+}
+
+// Next implements the Executor Next interface.
+func (e *ReloadDisabledOptimizeListExec) Next(ctx context.Context, _ *chunk.Chunk) error {
+	return LoadDisabledOptimizeList(e.ctx)
+}
+
+// LoadDisabledOptimizeList loads the latest data from table mysql.disabled_optimize_list.
+func LoadDisabledOptimizeList(ctx sessionctx.Context) (err error) {
```

Comment:
As we discussed in the paser pr, load disabled rule list should not load the disabled expr list anymore.